docs/recipes/amp-alertmanager-terraform.md

```diff
@@ -0,0 +1,141 @@
+You'll also need to change YOUR_ACCOUNT_ID in the service account section of the Kubernetes configuration to your AWS account ID.
+In this recipe, the ADOT Collector configuration uses an annotation (scrape=true) to tell which target endpoints to scrape. This allows the ADOT Collector to distinguish the sample app endpoint from kube-system endpoints in your cluster. You can remove this from the re-label configurations if you want to scrape a different sample app.
+5. Enter the following command to deploy the ADOT collector.
+```
+kubectl apply -f eks-prometheus-daemonset.yaml
+```
+
+### Deploying the Terraform module to configure Amazon Managed service for Prometheus workspace, recording rules & alert manager
+
+Now, we will  provision a Amazon Managed service for Prometheus workspace and will define an alerting rule that causes the Alert Manager to send a notification if a certain condition (defined in expr) holds true for a specified time period (for). Code in the Terraform language is stored in plain text files with the .tf file extension. There is also a JSON-based variant of the language that is named with the .tf.json file extension.
+
+```
+export SNS_TOPIC=<your SNS topic ARN>
+cat << EOF > main.tf
+provider "aws" {
+  profile    = "default"
+  region     = "$AWS_REGION"
+}
+
+resource "aws_prometheus_workspace" "amp-terraform-ws" {
+  alias = "amp-terraform-ws"
+}
+
+resource "aws_prometheus_rule_group_namespace" "amp-terraform-ws" {
+  name         = "rules"
+  workspace_id = aws_prometheus_workspace.amp-terraform-ws.id
+  data         = <<EOF
+groups:
+  - name: test
+    rules:
+    - record: metric:recording_rule
+      expr: rate(adot_test_counter0[5m])
+  - name: alert-test
+    rules:
+    - alert: metric:alerting_rule
+      expr: rate(adot_test_counter0[5m]) > 0.014
+      for: 5m    
+EOF
+}
+
+resource "aws_prometheus_alert_manager_definition" "amp-terraform-ws" {
+  workspace_id = aws_prometheus_workspace.amp-terraform-ws.id
+  definition   = <<EOF
+alertmanager_config: |
+  route:
+    receiver: 'default'
+  receivers:
+    - name: 'default'
+      sns_configs:
+      - topic_arn: $SNS_TOPIC
+        sigv4:
+          region: $AWS_REGION
+        attributes:
+          key: severity
+          value: SEV2
+EOF
+}
+```
+
+Once the file is created, execute the below commands to provision the resource using terraform:
+
+```
+terraform init
+terraform plan
+terraform apply
+```
+
+Once the above steps are complete, verify the setup end-to-end by using awscurl and query the endpoint. Look for the metric “metric:recording_rule”, and, if you successfully find the metric, then you’ve successfully created a recording rule:
+
+```
+awscurl https://aps-workspaces.us-east-1.amazonaws.com/workspaces/$WORKSPACE_ID/api/v1/rules  --service="aps"
+awscurl https://aps-workspaces.us-east-1.amazonaws.com/workspaces/$WORKSPACE_ID/alertmanager/api/v2/alerts --service="aps" -H "Content-Type: application/json"
+```
+
+## Clean up
```

Comment:
Remove this section, it's a dupe (see below)

Reply:
Cleared the section